tests/mr/device/limiting_mr_tests.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <rmm/detail/error.hpp>
+#include <rmm/device_buffer.hpp>
+#include <rmm/mr/device/limiting_resource_adaptor.hpp>
+
+#include <gtest/gtest.h>
+
+#define MB << 20
```

Comment:
This is just asking for problems. Use a user-defined suffix instead.

Reply:
https://github.com/rapidsai/rmm/blob/cce9081a7a72679b3fbac100d404ede6b4df1788/tests/mr/device/mr_test.hpp#L61-L66